go.mod

```diff
@@ -242,6 +261,14 @@ require (
 )
 
 //Note: add tag for go-ethereum before cutting hard-fork release
-replace github.com/ethereum/go-ethereum => github.com/iotexproject/go-ethereum v0.5.0
+replace github.com/ethereum/go-ethereum => github.com/iotexproject/go-ethereum v1.7.4-0.20250102060848-adaefd28e52f
 
 replace golang.org/x/xerrors => golang.org/x/xerrors v0.0.0-20190212162355-a5947ffaace3
+
+replace github.com/iotexproject/go-pkgs => github.com/iotexproject/go-pkgs v0.1.15-0.20250102061825-4556d014fc54
+
+replace github.com/ledgerwatch/erigon => github.com/erigontech/erigon v1.9.7-0.20240710071432-72ab70be8b28
```

Comment:
is it possible to directly use `import github.com/erigontech/erigon` in code, so no need to replace here?
same for L272

Reply:
yeah, I will update

Reply:
also rebase to latest